client.go

```diff
@@ -386,22 +424,28 @@ func (c *client) attemptConnection() (net.Conn, byte, bool, error) {
 		}
 		if err != nil {
 			ERROR.Println(CLI, err.Error())
+			c.logger.Error("Failed to connect to broker", slog.String("error", err.Error()), componentAttr(CLI))
 			WARN.Println(CLI, "failed to connect to broker, trying next")
+			c.logger.Warn("failed to connect to broker, trying next", componentAttr(CLI))
```

Comment:
Is this needed when there is an Error for the same thing? (think we want one or the other, not both)